components/sections/AboutMe.tsx

```diff
@@ -25,7 +25,7 @@ export const AboutMe = () => {
                             {content}
                         </ReactMarkdown>
                     </div>
-                    <div className="relative h-[25rem] w-[25rem] self-center overflow-hidden overflow-clip rounded-full md:h-[30rem] md:w-[30rem] lg:block">
+                    <div className="relative h-[20rem] w-[20rem] self-center overflow-hidden overflow-clip rounded-full md:h-[30rem] md:w-[30rem] lg:block">
```

Comment:
Use tailwind `h-80` instead of manual rem value: https://tailwindcss.com/docs/height